www/modules/custom/borg_project_metrics/borg_project_metrics.module

```diff
@@ -84,4 +90,284 @@ function borg_project_metrics_cron() {
       }
     }
   }
+
+  // Get and store backdrop_project_metrics top level data weekly.
+  if (date('G') == 24) {
+    if (date('l') == 'Monday') {
+      try {
+        $last_date = _get_display_metric_from_db('commits')['date'];
+        _parse_metrics('commits', $last_date);
+      }
+      catch(Exception $e) {
+        $message = $e->getMessage();
+        watchdog('borg_project_metrics', $message, array(), WATCHDOG_ERROR);
+      }
+    }
+
+    if (date('l') == 'Tuesday') {
+      try {
+        $last_date = _get_display_metric_from_db('comments')['date'];
+        _parse_metrics('comments', $last_date);
+      }
+      catch(Exception $e) {
+        $message = $e->getMessage();
+        watchdog('borg_project_metrics', $message, array(), WATCHDOG_ERROR);
+      }
+    }
+  }
+}
+
+/**
+ * Get the number of commits to Backdrop since 15 January 2015.
+ *
+ * @param string $type
+ * String to indicate what metric to get: {users, commits, comments}.
+ *
+ * @param string $date_time
+ * The date from DB borg_project_metrics table to set as since for the get.
+ *
+ * @return array $metrics
+ * Returns an array of json objects each element is a paginated response from the
+ * github api.
+ *
+ * @see _get_contrib_comments()
+ */
+function _get_metrics(string $type, string $date_time = '2017-06-01T00:00:00Z') {
+  global $settings;
+  $token = $settings['githubapi_token'];
+  $authorization = "Authorization: token $token";
+  $metrics = [];
+  $i = 1;
+  $date_time = str_replace(' ', 'T', $date_time) . 'Z';
```

Comment:
Not sure if this would cause the off-by-27 error, but it's the only thing I found just by looking at the code. The only other potential issue I can see is a namespace conflict with some of the internal helper functions.